core/src/main/java/org/jboss/pnc/rex/facade/TaskProviderImpl.java

```diff
@@ -103,6 +103,14 @@ public void cancel(String taskName) {
         controller.setMode(taskName, Mode.CANCEL);
     }
 
+    @Override
+    @Transactional
+    public void cancelByCorrelationID(String correlationID) {
```

Comment:
what should be the behavior if the correlationID doesn't exist? 404? No content?

Reply:
404 Sounds like a good answer.